pod.go

```diff
@@ -1161,6 +1167,44 @@ func togglePausePod(podID string, pause bool) (*Pod, error) {
 	return p, nil
 }
 
+func (p *Pod) handleDevices() error {
```

Comment:
Can we please call that one `attachDevices()` ? And this one should go through all its containers and call into an `Container.attachDevices()` as well.